spec/components/game_board_component_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe GameBoardComponent, type: :component do
+  #pending "add some examples to (or delete) #{__FILE__}"
+  let(:game) {
+    Game.new(passcode: %w(red red green green).to_json)
+  }
+  let(:attempts) {
+    [
+      Attempt.new(values: %w(blue yellow orange purple)),
+      Attempt.new(values: %w(blue yellow orange red))
+    ]
+  }
+  let(:feedback) { GameBoardComponent.new(game: game, attempts: attempts )}
+
+  it "renders game board elements" do
+    render_inline feedback
+
+    aggregate_failures do
+      expect(page.find(".secret-row")).to have_text "????", exact: true
+      expect(page.all(".guess-chances").map(&:text)).to eq (1..10).to_a.map(&:to_s).reverse
+
+      expect(page.find(".color-picker").find(".bg-red")["data-color"]).to eq "red"
```

Comment:
`data-*` attributes generally have no effect on what's rendered, so `data-color="red"` isn't the best assertion to test for.
I'd recomment using [has_selector?](https://rubydoc.info/github/teamcapybara/capybara/master/Capybara/Node/Matchers#has_selector%3F-instance_method) method instead, for example:
```ruby
expect(page.find(".color-picker")).to have_selector?(".bg-red")
```